native/src/components/SettingsSwitch.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react'
+import { Switch } from 'react-native'
+
+import { ThemeType } from 'build-configs/ThemeType'
+
+type PropsType = {
+  theme: ThemeType
+  onPress?: () => void
```

Comment:
Is it possible/necessary that `onPress` is `undefined`?

Reply:
Indeed...it would even be an error it `onPress` is `undefindend`